mobile/src/main/java/org/openhab/habdroid/util/ExtensionFuncs.kt

```diff
@@ -85,16 +85,17 @@ fun String.obfuscate(clearTextCharCount: Int = 3): String {
     return substring(0, clearTextCharCount) + "*".repeat(length - clearTextCharCount)
 }
 
-fun String?.toNormalizedUrl(): String {
+fun String?.toNormalizedUrl(): String? {
     return try {
-        val url = URL(orEmpty())
-            .toString()
+        val url = toString()
```

Comment:
`toString()` for a string? ;-)
Just use `this`.

Reply:
Fixed :)